packages/datadog-plugin-aerospike/src/index.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const { storage } = require('../../datadog-core')
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+
+class AerospikePlugin extends DatabasePlugin {
+  static get id () { return 'aerospike' }
+  static get operation () { return 'command' }
+  static get system () { return 'aerospike' }
+
+  bindStart (ctx) {
+    const { commandName, commandArgs } = ctx
+    const resourceName = commandName.slice(0, commandName.indexOf('Command'))
+    const store = storage.getStore()
+    const childOf = store ? store.span : null
+    let ns, set, key, bin, index
+    let meta = {}
+
+    if (resourceName.includes('Index')) {
+      [ns, set, bin, index] = commandArgs
+      meta = getMetaForIndex(ns, set, bin, index)
+    } else if (resourceName === 'Query') {
+      const { ns: queryNs, set: querySet } = commandArgs[2]
+      meta = getMetaForQuery({ ns: queryNs, set: querySet })
```

Comment:
Why rename these to something else only to map them back in the object construction?

```suggestion
      const { ns, set } = commandArgs[2]
      meta = getMetaForQuery({ ns, set })
```